test/serdes_test.c

```diff
@@ -554,11 +742,22 @@ KEYS(RSA);
 IMPLEMENT_TEST_SUITE(RSA, "RSA")
 KEYS(RSA_PSS);
 IMPLEMENT_TEST_SUITE(RSA_PSS, "RSA-PSS")
+IMPLEMENT_TEST_SUITE_MS(RSA, "RSA")
 
 int setup_tests(void)
 {
     int ok = 1;
 
+#ifndef OPENSSL_NO_DSA
+    size_t qbits = 160;          /* PVK only tolerates 160 Q bits */
```

Comment:
This size pretty much means its completely legacy..

Reply:
Oh, it is!  However, unless we decide to dump PVK and MSBLOB files, we'd better make sure it works; if for nothing else, as long as the STORE 'file:' loader recognises and loads such files...